hiera/data/cloudinit.yaml

```diff
@@ -13,3 +13,5 @@ public_address: "%{ipaddress_enp0s9}"
 
 yum_base_mirror: 'http://centos.mirror.uber.com.au'
 yum_epel_mirror: 'http://fedora.mirror.uber.com.au/epel'
+
+allowed_hosts: 192.168.242.%
```

Comment:
This should go in 'common'

'cloudinit' is for passing things in via metadata. The heat version will overwrite this file with what it finds in metadata, and the vagrant version will use this file since it doesn't have metadata.